avocado/utils/software_manager/backends/yum.py

```diff
@@ -111,16 +111,20 @@ def remove(self, name):
         except process.CmdError:
             return False
 
-    def add_repo(self, url):
+    def add_repo(self, url, opt_params={}):
```

Comment:
As the CI check error show, it is dangerous to use a dict as the default arguments of a function. 
Suggest using `**kwargs`.

Reply:
That's good point, thanks! 

It is true, we call the `add_repo` method only once (and nobody currently uses `opt_params`), so it should be safe for our use case from https://github.com/autotest/tp-qemu/pull/3550, but this is definitely thing to be avoided. (Fixed in both PRs.)